src/components/Button/Button.tsx

```diff
@@ -57,6 +61,10 @@ interface ButtonProps {
    * The size of the button.
    */
   size?: 'sm' | 'md' | 'lg';
+  /**
+   * The color variant of the button
+   */
+  variant?: 'primary' | 'success' | 'danger' | 'light' | 'dark' | 'white' | 'dark';
```

Comment:
Remove white and black

```suggestion
  variant?: 'primary' | 'success' | 'danger' | 'light' | 'dark';
```